home/api/Connectivity.ts

```diff
@@ -22,7 +22,7 @@ class Connectivity {
 
     try {
       const netInfo = await NetInfo.fetch();
-      this._isAvailable = netInfo.isConnected;
+      this._isAvailable = netInfo.isConnected ?? false;
```

Comment:
What's the API contract of `isConnected`?

Reply:
the initial state is `null` now - https://github.com/react-native-netinfo/react-native-netinfo/pull/444 - this represents an unknown/initialized state